nautobot/extras/tests/integration/test_configcontextschema.py

```diff
@@ -0,0 +1,69 @@
+from nautobot.utilities.testing.integration import SplinterTestCase
+
+
+class ConfigContextSchemaTestCase(SplinterTestCase):
+    """
+    Integration tests for the ConfigContextSchema model
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.user.is_superuser = True
+        self.user.save()
+        self.login(self.user.username, self.password)
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    def test_create_valid_config_context_schema(self):
+        """
+        Given a clean slate, navigate to and fill out the form for a valid schema object
+        Assert the object is successfully created
+        And the user is redirected to the detail page for the new object
+        """
+        # Navigate to ConfigContextSchema list view
+        self.browser.visit(self.live_server_url)
+        self.browser.links.find_by_partial_text("Extensibility").click()
+        self.browser.links.find_by_text("Config Context Schemas").click()
+
+        # Click add add button
+        self.browser.find_by_xpath("/html/body/div/div[1]/a").click()
+
+        # Fill out form
+        self.browser.fill("name", "Integration Schema 1")
+        self.browser.fill("description", "Description")
+        self.browser.fill("data_schema", '{"properties": {"a": {"type": "string"}}}')
+        self.browser.find_by_text("Create").click()
+
+        # Verify form redirect
+        self.assertTrue(self.browser.is_text_present("Created config context schema Integration Schema 1"))
+        self.assertTrue(self.browser.is_text_present("Clone"))
+
+    def test_create_invalid_config_context_schema(self):
+        """
+        Given a clean slate, navigate to and fill out the form for an invalid schema object
+        Provide normal details and an invalid JSON schema
+        Assert a validation error is raised
+        And the user is returned to the form
+        And the error details are listed
+        And the form is populated with the user's previous input
+        """
+        # Navigate to ConfigContextSchema list view
+        self.browser.visit(self.live_server_url)
+        self.browser.links.find_by_partial_text("Extensibility").click()
+        self.browser.links.find_by_text("Config Context Schemas").click()
+
+        # Click add add button
+        self.browser.find_by_xpath("/html/body/div/div[1]/a").click()
```

Comment:
Same as the other one: It appears you copied the Xpath from developer tools, but this isn't quite readable. Should we find the button by text instead?